pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
```

Comment:
We might better wait on this addition until we convert the repo (including any CI workflows) to use the new packaging methods.

Reply:
This is also not working properly yet. It works fine when we do `pip install -e .`. It fails, however, when we simply install  with `pip install .`, because python copies the tools as a module `dev_tools` into the site-packages. This breaks the relation to the `.nxdl.xml` and `.xsd` files. One solution could be to copy everything into a subfolder and make this a proper python module. This is typically how it's done with the `pyproject.toml` packaging. Maybe it also works to just introduce a top-level module which is copied to site-packages. Anyways, using this as a standalone module still needs some work and polish.